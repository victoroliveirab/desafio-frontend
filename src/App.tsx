import { BrowserRouter } from 'react-router-dom';
import AuthProvider from 'shared/providers/auth';
import Pages from 'pages';
import Header from 'components/Header';

function App() {
  return (
    <BrowserRouter>
      <AuthProvider>
        <Header />
        <Pages />
      </AuthProvider>
    </BrowserRouter>
  );
}

export default App;
