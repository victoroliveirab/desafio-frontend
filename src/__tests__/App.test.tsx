import App from 'App';
import { render } from './utils';

describe('<App />', () => {
  test('should match snapshot', () => {
    const component = render(<App />);

    expect(component).toMatchSnapshot();
  });
});
