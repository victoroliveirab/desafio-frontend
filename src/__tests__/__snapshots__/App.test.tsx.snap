// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        data-auto_select="true"
        data-callback="ptLoginCallback"
        data-cancel_on_tap_outside="false"
        data-context="signin"
        data-moment_callback="ptNotifyCallback"
        data-ux_mode="popup"
        id="g_id_onload"
      />
      <div
        id="g_id_signin"
        style="width: 200px; position: absolute; right: 40px; top: 20px;"
      />
      <div>
        not logged in
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      data-auto_select="true"
      data-callback="ptLoginCallback"
      data-cancel_on_tap_outside="false"
      data-context="signin"
      data-moment_callback="ptNotifyCallback"
      data-ux_mode="popup"
      id="g_id_onload"
    />
    <div
      id="g_id_signin"
      style="width: 200px; position: absolute; right: 40px; top: 20px;"
    />
    <div>
      not logged in
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
